openwisp_radius/saml/views.py

```diff
@@ -52,6 +53,10 @@ class AssertionConsumerServiceView(
 ):
     def post_login_hook(self, request, user, session_info):
         """If desired, a hook to add logic after a user has successfully logged in."""
+        # In some cases, it possible that the organization cache for
+        # the user is not updated before execution of the following
+        # code. Hence, the cache is manually updated here.
+        cache.delete('user_{}_organizations'.format(user.pk))
```

Comment:
@nemesisdesign the change that we did in openwisp-users 1.0.2 is not useful here because `openwisp-users.apps.OpenwispUsersConfig.update_organizations_dict` expects either an `OrganizationUser` or `OrganizationOwner` object. Therefore, I have used cache.delete here.